### PR TITLE
cmd/atlas-action: init atlas action dispatcher cmdline

### PR DESCRIPTION
**Pull request by rotemtam**

### CODE REVIEW

File: cmd/atlas-action/main.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		action.Fatalf("Failed to run command: %s", err)
+	}
+}
+
+// RunAction is a command to run one of the Atlas GitHub Actions.
+type RunAction struct {
+	Action string `help:"Command to run" required:""`
+}
+
+func (r *RunAction) Run(_ context.Context, _ *atlasexec.Client, _ *githubactions.Action) error {
```
Comment (a8m) [inline cmd/atlas-action/main.go:37]: Another way it to let packages register themselves and empty import.

**Review by a8m — APPROVED** [review]

File: cmd/atlas-action/main.go (hunk 1 above)
Comment (a8m) [reply cmd/atlas-action/main.go:37]: Just an idea, not a suggestion

Comment (rotemtam) [reply cmd/atlas-action/main.go:37]: When this gets large enough , initially we will have 3-4 opts so no big deal

File: .github/workflows/ci-go.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      - master
+  pull_request:
+jobs:
+  golangci-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v4
+        with:
+          go-version: '1.21'
```
Comment (dorav) [inline .github/workflows/ci-go.yaml:14]: You can use the go version from the mod file, instead of 'go-version', it's a different flag 'go-version-file'

**Pull request merged**